client/src/components/tcg-logo/tcg-logo.tsx

```diff
@@ -1,12 +1,14 @@
 import css from './tcg-logo.module.css'
+// @ts-ignore
+import {CONFIG} from '../../../../config'
```

Comment:
We should avoid importing config directly to frontend, can I ask you to use environment variable using the `define` config in `vite.config.js` (also you will need to add type in `global.d.ts`). We already to the same for PORT & LIMITS.

Reply:
Thanks for the advice
Applied in 6dd2cef